examples/basic-host/package.json

```diff
@@ -23,7 +23,6 @@
     "@types/react": "^19.2.2",
     "@types/react-dom": "^19.2.2",
     "@vitejs/plugin-react": "^4.3.4",
-    "bun": "^1.3.2",
```

Comment:
Is this dependency causing a problem on Windows?

In 9f8d00ef8a9953bd09a4fbdc1f91b294d0ba3714, I explicitly added `bun` as a dependency to the examples in order to improve on-ramping (users only have to run `npm install`).